### ISSUE TITLE
Missing chat locked warning

### ISSUE THREAD

**Issue by PhMemmel** [bigbluebutton/bigbluebutton#12014]
**Describe the bug**
If a private chat is opened by a locked user and private chat is disabled there usually is a red warning being shown: "Chat is locked, messages can't be sent"

**To Reproduce**
1. Open meeting, join a moderator and at least two attendees
2. Attendee1 opens private chat to Attendee2, writes something
3. Moderator locks private chatting
4. Attendee1: click on Attendee2 in user list -> start private chat, old private chat including old messages reappears (I'm not sure if this is intended, but would be another issue and it's the same behavior as in 2.2)
5. Chat is locked (as it should), but the warning isn't shown

**Additional context**

- The warning **IS** being shown if i close the chat (small cross) so it disappears above the user list.
- The warning **IS** being shown when I don't close the chat, but just click the chat on the private chat when it's still available above the user list.

**Expected behavior**
Warning should also be shown in the described scenario.

**Actual behavior**
See reproduction steps.

**Screenshots**
Location to click:
![grafik](https://user-images.githubusercontent.com/65113153/114600338-84151880-9c94-11eb-9ca6-ca06f7cf5027.png)
What it looks like:
![grafik](https://user-images.githubusercontent.com/65113153/114600530-be7eb580-9c94-11eb-8dd4-066e86884c81.png)
What it's supposed to look like:
![grafik](https://user-images.githubusercontent.com/65113153/114600579-cdfdfe80-9c94-11eb-9494-482cd532489b.png)

**BBB version (optional):**
2.3-beta-3

**Desktop (please complete the following information):**
Tested on current Chrome, but probably browser independent

**Comment by ramonlsouza** [comment]
This issue can be closed. It is no longer reproducible (after #12013), as attendees can't start a private chat if it is locked by a moderator.

**Comment by PhMemmel** [comment]
I tested it in my dev environment after pulling current develop and can confirm, it's fixed.

@ramonlsouza Your explanation seems to be wrong though. This is still possible:

- Attendee1 starts a private chat with attendee2 while private chatting isn't locked by mod.
- Mod locks private chats
- attendee1 still can click on attendee2 in user list and start private chat because it already existed before.

However, the warning is now shown properly in this case as well. This probably is due to your work in PR #12027, thank you very much for that! The PR #12013 you mentioned hasn't touched that problem I think.

**Comment by ramonlsouza** [comment]
> I tested it in my dev environment after pulling current develop and can confirm, it's fixed.
> 
> @ramonlsouza Your explanation seems to be wrong though. This is still possible:
> 
> * Attendee1 starts a private chat with attendee2 while private chatting isn't locked by mod.
> * Mod locks private chats
> * attendee1 still can click on attendee2 in user list and start private chat because it already existed before.
> 
> However, the warning is now shown properly in this case as well. This probably is due to your work in PR #12027, thank you very much for that! The PR #12013 you mentioned hasn't touched that problem I think.

Yes, by "start a private chat" I mean open a private chat for the first time in that meeting (this was the only condition where the warning would not appear).
